SwrveSDKCommon/src/main/java/com/swrve/sdk/ISwrveCommon.java

```diff
@@ -0,0 +1,52 @@
+package com.swrve.sdk;
+
+import android.content.Context;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Map;
+
+public interface ISwrveCommon
+{
+    String getApiKey();
+
+    String getSessionKey();
+
+    short getDeviceId();
+
+    int getAppId();
+
+    String getUserId();
+
+    String getAppVersion();
+
+    String getUniqueKey();
+
+    String getBatchURL();
+
+    String getCachedLocationData();
```

Comment:
Think the location specific stuff could be pulled into separate interface perhaps?  (getCachedLocationData & setLocationVersion). Perhaps a ISwrveLocationSDK